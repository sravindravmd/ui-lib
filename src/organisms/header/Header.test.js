import React from 'react'
import { render, cleanup } from '@testing-library/react'
import 'regenerator-runtime/runtime'
import { Header } from './'
import links from './mock'

afterEach(cleanup)

const props = {
  company: 'SPORTS & CO',
  productLinks: links,
}

describe('<Global Header/>', () => {
  it('renders', () => {
    const { container } = render(<Header {...props} />)
    expect(container).toMatchSnapshot()
  })
})
