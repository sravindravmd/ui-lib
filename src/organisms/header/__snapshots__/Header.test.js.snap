// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Global Header/> renders 1`] = `
<div>
  <header
    class="sc-bdfBwQ cytjPI"
  >
    <div
      class="inner"
    >
      <div
        class="col left"
      >
        <div
          class="logo"
        >
          SPORTS & CO
        </div>
      </div>
      <div
        class="col middle"
      >
        <div
          class="product-links"
        >
          <div
            class="product-link "
          >
            <a
              href="/overview"
              title="Overview"
            >
              <span
                class="icon"
              >
                <svg
                  fill="none"
                  height="24"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <ellipse
                    cx="19"
                    cy="11.5504"
                    fill="#121213"
                    rx="2"
                    ry="1.9"
                  />
                  <ellipse
                    cx="12"
                    cy="11.5504"
                    fill="#121213"
                    rx="2"
                    ry="1.9"
                  />
                  <ellipse
                    cx="5"
                    cy="11.5504"
                    fill="#121213"
                    rx="2"
                    ry="1.9"
                  />
                  <ellipse
                    cx="12"
                    cy="4.9"
                    fill="#121213"
                    rx="2"
                    ry="1.9"
                  />
                  <ellipse
                    cx="12"
                    cy="18.2008"
                    fill="#121213"
                    rx="2"
                    ry="1.9"
                  />
                </svg>
              </span>
              <span
                class="text"
              >
                Overview
              </span>
            </a>
          </div>
          <div
            class="product-link active"
          >
            <a
              href="/pim"
              title="PIM"
            >
              <span
                class="icon"
              >
                <svg
                  fill="none"
                  height="18"
                  viewBox="0 0 18 18"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00702339 0.999975 1.80947 1.70756)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00739302 0.999973 1.8101 8.53764)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00739302 0.999973 1.8101 15.3677)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00702339 0.999975 8.99941 1.70756)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00739302 0.999973 9.00004 8.53764)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00739302 0.999973 9.00004 15.3677)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00702339 0.999975 16.1894 1.70756)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00739302 0.999973 16.19 8.53764)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.79748"
                    ry="1.70761"
                    transform="matrix(1 0 0.00739302 0.999973 16.19 15.3677)"
                  />
                </svg>
              </span>
              <span
                class="text"
              >
                PIM
              </span>
            </a>
          </div>
          <div
            class="product-link "
          >
            <a
              href="/xpm"
              title="XPM"
            >
              <span
                class="icon"
              >
                <svg
                  fill="none"
                  height="24"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <ellipse
                    fill="#121213"
                    rx="1.65093"
                    ry="1.60102"
                    transform="matrix(1 0 0.00724234 0.999974 15.6679 8.01113)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.65093"
                    ry="1.60102"
                    transform="matrix(1 0 0.00688024 0.999976 19.3358 4.60098)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.60102"
                    ry="1.65093"
                    transform="matrix(-4.50741e-08 1 -0.999977 0.00681102 15.6562 14.8447)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.60102"
                    ry="1.65093"
                    transform="matrix(-4.50741e-08 1 -0.999977 0.00681102 19.3247 18.4873)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.60102"
                    ry="1.65093"
                    transform="matrix(-4.50741e-08 1 -0.999977 0.00681102 11.9228 11.4345)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.65093"
                    ry="1.60102"
                    transform="matrix(-1 0 -0.00724234 0.999974 8.33015 8.01113)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.65093"
                    ry="1.60102"
                    transform="matrix(-1 0 -0.00688024 0.999976 4.66178 4.60098)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.60102"
                    ry="1.65093"
                    transform="matrix(4.50741e-08 1 0.999977 0.00681102 8.34181 14.8447)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.60102"
                    ry="1.65093"
                    transform="matrix(4.50741e-08 1 0.999977 0.00681102 4.67335 18.4873)"
                  />
                </svg>
              </span>
              <span
                class="text"
              >
                XPM
              </span>
            </a>
          </div>
          <div
            class="product-link "
          >
            <a
              href="/offers"
              title="OFFERS"
            >
              <span
                class="icon"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 18 20"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739303 0.999973 9.00013 9.72827)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739303 0.999973 5.23792 5.36108)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739303 0.999973 5.23792 13.699)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739303 0.999973 12.7618 13.699)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739303 0.999973 12.7618 5.36108)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739299 0.999973 16.1056 9.72827)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739299 0.999973 1.89417 9.72827)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00739299 0.999973 9.00013 17.2732)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.88096"
                    ry="1.78691"
                    transform="matrix(1 0 0.00702336 0.999975 8.99946 1.78687)"
                  />
                </svg>
              </span>
              <span
                class="text"
              >
                OFFERS
              </span>
            </a>
          </div>
          <div
            class="product-link "
          >
            <a
              href="/oms"
              title="OMS"
            >
              <span
                class="icon"
              >
                <svg
                  fill="none"
                  height="18"
                  viewBox="0 0 18 18"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 1.59474e-07 0.00737265 0.995702 8.61748 8.90062)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 1.59474e-07 0.00737265 0.995702 16.2669 8.90062)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 0 0.00737247 0.995702 5.17509 12.1682)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 0 0.00737247 0.995702 12.8245 12.1682)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 0 0.00737247 0.995702 1.7332 15.803)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 0 0.00737247 0.995702 9.38261 15.803)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 -7.9395e-08 0.00737238 0.995702 5.17509 5.26781)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 -7.9395e-08 0.00737238 0.995702 12.8245 5.26781)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 8.42937e-08 0.00700395 0.995705 1.73259 1.635)"
                  />
                  <ellipse
                    fill="#121213"
                    rx="1.7138"
                    ry="1.64206"
                    transform="matrix(1.00425 8.42937e-08 0.00700395 0.995705 9.38201 1.635)"
                  />
                </svg>
              </span>
              <span
                class="text"
              >
                OMS
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="col actions"
      >
        <div
          class="sc-gsTCUz aHWJs"
        >
          <div
            class="sc-jSgupP cITbau  notification bottom"
          >
            <div
              class="icon-notify red-dot"
            >
              <svg
                fill="none"
                height="16"
                viewBox="0 0 24 16"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M0 4C0.109375 4 0.257812 4.00391 0.445312 4.01172C0.632812 4.01953 0.820312 4.02344 1.00781 4.02344H1.24219C1.3125 4.02344 1.375 4.01562 1.42969 4L22.5 0.25C22.5469 0.242187 22.6094 0.238281 22.6875 0.238281C22.7656 0.238281 22.8516 0.234375 22.9453 0.226562C23.1328 0.226562 23.3281 0.230469 23.5312 0.238281C23.7344 0.246094 23.8906 0.25 24 0.25V13.75H23.543C23.3477 13.75 23.1562 13.7539 22.9688 13.7617H22.6172C22.5156 13.7617 22.4336 13.7539 22.3711 13.7383L13.4297 12.1445C13.3516 12.5898 13.2031 13.0039 12.9844 13.3867C12.7656 13.7695 12.4883 14.0977 12.1523 14.3711C11.8164 14.6445 11.4492 14.8594 11.0508 15.0156C10.6523 15.1719 10.2188 15.25 9.75 15.25C9.23438 15.25 8.75 15.1523 8.29688 14.957C7.84375 14.7617 7.44531 14.4961 7.10156 14.1602C6.75781 13.8242 6.48828 13.4258 6.29297 12.9648C6.09766 12.5039 6 12.0156 6 11.5C6 11.2812 6.02344 11.0586 6.07031 10.832L1.42969 10C1.36719 9.99219 1.30078 9.98828 1.23047 9.98828C1.16016 9.98828 1.07812 9.98437 0.984375 9.97656C0.8125 9.97656 0.640625 9.98047 0.46875 9.98828C0.296875 9.99609 0.140625 10 0 10V4ZM9.75 13.75C10.0234 13.75 10.2852 13.7031 10.5352 13.6094C10.7852 13.5156 11.0078 13.3867 11.2031 13.2227C11.3984 13.0586 11.5625 12.8594 11.6953 12.625C11.8281 12.3906 11.918 12.1406 11.9648 11.875L7.53516 11.0898C7.51172 11.2305 7.5 11.3672 7.5 11.5C7.5 11.8125 7.55859 12.1055 7.67578 12.3789C7.79297 12.6523 7.95312 12.8906 8.15625 13.0937C8.35938 13.2969 8.59766 13.457 8.87109 13.5742C9.14453 13.6914 9.4375 13.75 9.75 13.75ZM22.5 1.76172L1.5 5.5V8.5C1.55469 8.5 1.59766 8.50391 1.62891 8.51172L22.5 12.2383V1.76172Z"
                  fill="white"
                />
              </svg>
            </div>
            <div
              class="content"
            >
              <div
                class="content-inner"
              >
                <span
                  class="arrows"
                />
                <div
                  class="tabs-notify"
                >
                  <div
                    class="sc-gKsewC aHCfK"
                  >
                    <div
                      class="sc-fubCfw eUGLku"
                    >
                      <div
                        class="sc-pFZIQ hdkyvP"
                      >
                        All
                      </div>
                      <div
                        class="sc-pFZIQ hxcPvT"
                      >
                        Mentions
                      </div>
                      <div
                        class="sc-pFZIQ hxcPvT"
                      >
                        Fabric Updates
                      </div>
                    </div>
                    <div
                      title=""
                    >
                      <div
                        class="sc-hKgILt dfrCJo"
                      >
                        <div
                          class="title"
                        >
                          Today
                        </div>
                        <div
                          class="sc-eCssSg huoVAZ"
                        >
                          <div
                            class="column left"
                          >
                            <div
                              class="image"
                            >
                              <img
                                alt=""
                                src=""
                              />
                            </div>
                            <div
                              class="description"
                            >
                              <p>
                                <span>
                                  The Mari Bag (0200388292)
                                </span>
                                 was assigned to you and needs your approval.
                              </p>
                            </div>
                          </div>
                          <div
                            class="column right"
                          >
                            <button
                              class="sc-jrAGrp gFuVLv"
                            >
                              Review item
                            </button>
                          </div>
                        </div>
                        <div
                          class="sc-eCssSg huoVAZ"
                        >
                          <div
                            class="column left"
                          >
                            <div
                              class="image"
                            >
                              <img
                                alt=""
                                src=""
                              />
                            </div>
                            <div
                              class="description"
                            >
                              <p>
                                <span>
                                  The Mari Bag (0200388292)
                                </span>
                                 was assigned to you and needs your approval.
                              </p>
                            </div>
                          </div>
                          <div
                            class="column right"
                          >
                            <button
                              class="sc-jrAGrp gFuVLv"
                            >
                              Review item
                            </button>
                          </div>
                        </div>
                      </div>
                      <div
                        class="sc-hKgILt dfrCJo"
                      >
                        <div
                          class="title"
                        >
                          Yesterday
                        </div>
                        <div
                          class="sc-eCssSg huoVAZ"
                        >
                          <div
                            class="column left"
                          >
                            <div
                              class="image"
                            >
                              <img
                                alt=""
                                src=""
                              />
                            </div>
                            <div
                              class="description"
                            >
                              <p>
                                <span>
                                  The Mari Bag (0200388292)
                                </span>
                                 was assigned to you and needs your approval.
                              </p>
                            </div>
                          </div>
                          <div
                            class="column right"
                          >
                            <button
                              class="sc-jrAGrp gFuVLv"
                            >
                              Review item
                            </button>
                          </div>
                        </div>
                        <div
                          class="sc-eCssSg huoVAZ"
                        >
                          <div
                            class="column left"
                          >
                            <div
                              class="image"
                            >
                              <img
                                alt=""
                                src=""
                              />
                            </div>
                            <div
                              class="description"
                            >
                              <p>
                                <span>
                                  The Mari Bag (0200388292)
                                </span>
                                 was assigned to you and needs your approval.
                              </p>
                            </div>
                          </div>
                          <div
                            class="column right"
                          >
                            <button
                              class="sc-jrAGrp gFuVLv"
                            >
                              Review item
                            </button>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-kEjbxe brIpAs"
        >
          <div
            class="sc-jSgupP cITbau  profile bottom"
          >
            <div
              class="user-actions"
            >
              <div
                class="avatar"
              />
              <div
                class="user-name"
              >
                Tiffany Doe
              </div>
            </div>
            <div
              class="content"
            >
              <div
                class="content-inner"
              >
                <span
                  class="arrows"
                />
                <div
                  class="sc-iqHYGH kqfzBI"
                >
                  <div
                    class="header"
                  >
                    <div
                      class="avatar"
                    >
                      <span />
                    </div>
                    <div
                      class="title"
                    >
                      <strong>
                        Tiffany Doe
                      </strong>
                      <p>
                        tiffany.doe@jcrew.com
                      </p>
                    </div>
                  </div>
                  <ul>
                    <li>
                      <a
                        href=""
                      >
                        Edit Profile
                      </a>
                    </li>
                    <li>
                      <a
                        href=""
                      >
                        Preferences
                      </a>
                    </li>
                    <li>
                      <a
                        href=""
                      >
                        Logout
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </header>
</div>
`;
